src/BlockTypes/ProductPrice.php

```diff
@@ -86,9 +86,9 @@ protected function render( $attributes, $content, $block ) {
 			$text_align_styles_and_classes = StyleAttributesUtils::get_text_align_class_and_style( $attributes );
 
 			return sprintf(
-				'<div class="wc-block-components-product-price wc-block-grid__product-price %1$s %2$s" style="%3$s">
+				'<div class="wp-block-woocommerce-product-price"><div class="wc-block-components-product-price wc-block-grid__product-price %1$s %2$s" style="%3$s">
```

Comment:
Is it necessary to add one more div? Can we not add the `wp-block-woocommerce-product-price` class to the existing div instead? 🤔

Reply:
I don't have a strong opinion, I just followed the structure we have in the `edit.tsx` document:

https://github.com/woocommerce/woocommerce-blocks/blob/trunk/assets/js/atomic/blocks/product-elements/price/edit.tsx#L92-L94

I guess both options would work, but removing the wrapping `<div>` would make the React code a bit more complex as we need to pass down the props we get from `useBlockProps()` two levels down.

Do you think the extra div is problematic? If so, I'm happy to change it.

Reply:
@Aljullu Thanks for explaining the rationale behind the extra `<div>` . I understand that removing it would make the React code more complex. Since it's not a blocker and doesn't affect the overall functionality, I agree with your decision to keep the extra `<div>` 🤝